core/modules/user/user.module

```diff
@@ -451,6 +451,26 @@ function user_is_blocked($name) {
 }
 
 /**
+ * Checks if a user has a role.
+ *
+ * @param int $rid
+ *   A role ID.
+ *
+ * @param bool|object $account
+ *   (optional) A user account. Defaults to the current user.
```

Comment:
This @param should not have leading new line. It also doesn't look like "bool" is a valid parameter type. Lastly, in Backdrop, we pass proper User objects, not a generic object, so the parameter line should be:
```
 + * @param User $account 
```